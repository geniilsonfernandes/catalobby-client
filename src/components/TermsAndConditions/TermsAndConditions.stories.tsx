import type { Meta, StoryObj } from '@storybook/react';
import TermsAndConditions from '.';

const meta: Meta<typeof TermsAndConditions> = {
  title: 'Components/TermsAndConditions',
  component: TermsAndConditions,
  parameters: {
    layout: 'centered'
  },
  tags: ['autodocs'],
  argTypes: {}
};

export default meta;
type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {}
};
