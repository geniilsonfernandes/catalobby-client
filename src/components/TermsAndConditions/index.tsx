const TermsAndConditions = () => {
  return (
    <p className="text-sm text-gray-400 ">
      Clicando em <b>Criar minha conta</b>, voce concorda com os{' '}
      <a className=" text-primary-100" href="/login">
        Termos de Servi√ßo
      </a>{' '}
      e os{' '}
      <a className=" text-primary-100" href="/login">
        {' '}
        politicas de privacidade.
      </a>
    </p>
  );
};

export default TermsAndConditions;
