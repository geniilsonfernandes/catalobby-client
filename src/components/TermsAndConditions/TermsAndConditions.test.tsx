import { render, screen } from '@testing-library/react';
import { describe, expect, test } from 'vitest';

import TermsAndConditions from '.';

describe('TermsAndConditions render', () => {
  test('should render correctly', () => {
    const { container } = render(<TermsAndConditions />);

    expect(container).toMatchSnapshot();
    expect(screen.getByText('Termos de Servi√ßo')).toBeInTheDocument();
    expect(screen.getByText('politicas de privacidade.')).toBeInTheDocument();
  });
});
