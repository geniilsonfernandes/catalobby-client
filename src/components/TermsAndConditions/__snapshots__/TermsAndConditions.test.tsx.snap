// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TermsAndConditions render > should render correctly 1`] = `
<div>
  <p
    class="text-sm text-gray-400 "
  >
    Clicando em 
    <b>
      Criar minha conta
    </b>
    , voce concorda com os
     
    <a
      class=" text-primary-100"
      href="/login"
    >
      Termos de Servi√ßo
    </a>
     
    e os
     
    <a
      class=" text-primary-100"
      href="/login"
    >
       
      politicas de privacidade.
    </a>
  </p>
</div>
`;
